backend/src/services/AccountManager.ts

```diff
@@ -1,62 +1,59 @@
-import prisma from '../utils/prismaHandler';
+import { PrismaClient } from '@prisma/client';
 import bcrypt from 'bcryptjs';
 import * as IAccountManager from './IAccountManager';
+import logger from '../utils/logger';
 
-/**
- * A service for accessing and editing the anonymous
- * voting app's users in the database.
- */
-const _prisma = prisma;
-
-export const CreateUser = async (
+export const createUser = async (
     email: string,
     password: string,
     username: string,
     firstName: string,
-    lastName: string
+    lastName: string,
+    prisma: PrismaClient
```

Comment:
Had to move to giving Prisma as parameter as the tests weren't working otherwise => can be fixed later if deemed necessary